datascience-notebook/Dockerfile

```diff
@@ -10,11 +10,18 @@ ENV CHOWN_HOME=no
 
 COPY ./files/apt_packages /tmp/
 
-RUN conda install boto3 \
+RUN conda install boto3 black flake8 jupyterlab_code_formatter \
     && apt-get update \
     && apt-get install -y $(cat /tmp/apt_packages) \
     && rm -rf /var/lib/apt/lists/*
 
+RUN npm install jupyterlab-flake8
+
+RUN jupyter labextension install @ryantam626/jupyterlab_code_formatter
+RUN jupyter serverextension enable --py jupyterlab_code_formatter
+
+RUN jupyter labextension install jupyterlab-flake8
```

Comment:
This is a lot of `RUN` layers. Can we combine them with `&&`?

Reply:
Or you could do a multi-stage Dockerfile and just copy the results of installing the formatter and flake8 into the final image

Reply:
Added the lines as one RUN command